models/reporting/olids/person_demographics/dim_person_demographics_snapshot_input.sql

```
@@ -0,0 +1,27 @@
+
+Exposes only stable / structural demographic match keys (for control-cohort
+matching and to track structural change like practice moves or death).
+`age` is deliberately EXCLUDED: it changes continuously and would version every
+person on their birthday. Track structure, not age.
+
+Downstream: snapshots/dim_person_demographics_snapshot.yml
+*/
+
+select
+    person_id
+    , gender
```

SQLFluff (4.2.2)

[error] Found leading comma ','. Expected only trailing near line breaks.

(LT04)